idea-plugin/src/main/kotlin/org/jetbrains/compose/desktop/ide/preview/PreviewStateService.kt

```diff
@@ -34,7 +34,7 @@ class PreviewStateService : Disposable {
             val frameConfig = FrameConfig(
                 width = c.width,
                 height = c.height,
-                scale = null
+                scale = c.graphicsConfiguration.defaultTransform.scaleX
```

Comment:
What will happen if we'll move IDE window to display with different DPI?

Reply:
I have no idea. @igordmn @Rsedaikin do you guys have any ideas, if this is a problem with `graphicsConfiguration.defaultTransform.scaleX` Swing API?

Reply:
Seems it should work, since swing should trigger resize event for all components (as @igordmn said during today's sync). I want to merge this PR now, will check DPI change later.

Reply:
I checked, it works